uber/config.py

```diff
@@ -73,9 +73,19 @@ def make_enum(self, enum_name, section, prices=False):
                 lookup[val] = desc
 
         enum_name = enum_name.upper()
-        setattr(self, enum_name + '_OPTS', opts)
-        setattr(self, enum_name + '_VARS', varnames)
-        setattr(self, enum_name + ('' if enum_name.endswith('S') else 'S'), lookup)
+        # When loading plugin's configs, we want to make sure we don't override existing enums
+        if hasattr(self, enum_name + '_OPTS'):
```

Comment:
Unfortunately, it probably needs to be even messier, in that there are a bunch of cases which should probably be handled if we're going to officially support having plugins define their own enum subsections which then get merged into the RAMS-defined enums.

The main thing we're not currently handling is what happens when someone defines an individual option within the enum which was already defined.
- We should only add the enum to ``_VARS`` if it wasn't already there.
- We should replace-in-place the tuple in ``_OPTS`` with the new value/description pair (the value will be the same but the description might be different).
- It's not obvious at a glance, but one of the behaviors of this function is that if you set an enum option's value to the empty string, it shows up in the ``_VARS`` list of attribute names but does NOT show up in either the dictionary or list-of-tuples.  This allows us to take options defined in ``configspec.ini`` and remove them as options.  This would not generally happen for things like badge types, but if an event wanted to get rid of a ribbon type they weren't using or something, they could configure their ``development.ini`` to set the description to the empty string and then it won't show up in the ribbons dropdown.  We should either handle this case (where a plugin specifies an empty string value) or detect and error on it.